src/aether-elasticsearch-consumer/values.yaml

```diff
@@ -11,3 +11,12 @@ app:
   esConsumerKafkaConfigPath: "/code/conf/consumer/kafka.json"
   command: "start"
   port: "9009"
+  secret: consumer-secrets
+
+kafka:
+  url: 
+  user: admin
```

Comment:
Is this a placeholder? CCloud doesn't allow logical names of users (which are keys), only service accounts, so there's no default value for this.

Reply:
removed this